src/models/proof_abstractions/tasm/prover_job.rs

```diff
@@ -14,6 +14,8 @@ use std::process::Stdio;
 use tasm_lib::maybe_write_debuggable_vm_state_to_disk;
 #[cfg(not(test))]
 use tokio::io::AsyncWriteExt;
+#[cfg(not(test))]
+use tracing::info;
```

Comment:
style: maybe just call `tracing::info!("...")` in the code, and we can avoid this use statement ugly `#[cfg(not(test))]` here.